config/environments/test.rb

```diff
@@ -36,6 +36,10 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # API location can be specified in the environment
+  # But defaults to the dev service
+  config.api_service_url = ENV['API_SERVICE_URL'] || 'https://lr-ppd-dev-pres.epimorphics.net'
```

Comment:
Per my message on Slack, there's no point defaulting this URL to a service that is not running. If we can't point to the new service yet, we should leave the default out